src/main/java/com/bullhornsdk/data/api/BullhornRestCredentials.java

```diff
@@ -82,10 +103,16 @@ public void setRestAuthorizeUrl(String restAuthorizeUrl) {
 		this.restAuthorizeUrl = restAuthorizeUrl;
 	}
 
+	public boolean credentialCheck() {
+	    return restUrl == "default" && bhRestToken == "default";
+    }
+
 	@Override
 	public int hashCode() {
 		final int prime = 31;
 		int result = 1;
+        result = prime * result + ((bhRestToken == null) ? 0 : bhRestToken.hashCode());
```

Comment:
minor nitpick - need to match the formatting of the repo - you're a space off

Reply:
Formatting is now in line.